src/main/java/net/logstash/logback/appender/AbstractLogstashTcpSocketAppender.java

```diff
@@ -621,9 +769,17 @@ public void setPort(int port) {
     public int getPort() {
         return port;
     }
+    
+    public void setHostAddresses(String hostAddresses) {
```

Comment:
Great! I like it as well. 
I was a bit worried about deprecating the host/port fields... We could still keep them and make the two options exclusive. Maybe with a warning at runtime and in the doc saying the host/port will be removed in future releases.